src/pages/BuySell/GemBuySell.tsx

```diff
@@ -0,0 +1,165 @@
+                      flexDirection={'column'}
+                      alignItems={'center'}
+                      marginTop={'15%'}
+                    >
+                      <Box>
+                        <Button>Continue</Button>
+                      </Box>
+                      <Box marginTop={'5%'}>
+                        <Button>Cancel</Button>
+                      </Box>
+                    </Flex>
+                  </TabPanel>
+                  <TabPanel>
+                    <Flex
+                      justifyContent={'flex-start'}
+                      flexDirection={'column'}
+                      alignItems={'flex-start'}
+                    >
+                      <Text lineHeight={1}>Assest to sell</Text>
+                      <Box marginTop={'5%'}>
+                        <Menu>
+                          <MenuButton as={Button} rightIcon={<ChevronDownIcon />}>
+                            Select an assest to sell
+                          </MenuButton>
+                          <MenuList>
+                            {location.state.sellCoinList.map(sellCoin => {
+                              return (
+                                <MenuItem>
+                                  <Box>
+                                    <Button
+                                      variant='ghost'
+                                      isActive={true}
+                                      justifyContent='flex-start'
+                                      _focus={{
+                                        shadow: 'outline-inset'
+                                      }}
+                                      padding={'25px'}
+                                    >
+                                      <Avatar
+                                        src={`https://gem-widgets-assets.s3-us-west-2.amazonaws.com/currencies/crypto/${sellCoin.ticker.toLowerCase()}.svg`}
+                                        bg={useColorModeValue('gray.200', 'gray.700')}
+                                      />
+                                      <Box textAlign='left'>
+                                        <Text lineHeight={1}>{sellCoin.name}</Text>
+                                        <Text fontWeight='normal' fontSize='sm'>
+                                          {sellCoin.ticker}
+                                        </Text>
+                                      </Box>
+                                    </Button>
+                                  </Box>
+                                </MenuItem>
+                              )
+                            })}
+                          </MenuList>
+                        </Menu>
+                      </Box>
+                    </Flex>
+                    <Flex
+                      justifyContent={'center'}
+                      flexDirection={'column'}
+                      alignItems={'center'}
+                      marginTop={'15%'}
```

Comment:
i think we have enums for these margin sizes defined so we're consistent with our styles everywhere